File: Csharp/CodeWars/src/Resolutions/BeginnerSeries3SumOfNumbers.cs
namespace CodeWars.Resolutions;

public class BeginnerSeries3SumOfNumbers
{
    public static int GetSum(int a, int b)
    {
        int init = Math.Min(a, b);
        int end = Math.Max(a, b);
        int sum = 0;

        for (int i = init; i <= end; i++)
            sum += i;
        return sum;
    }
}

File: TypeScript/Codewars/src/resolutions/Beginner-Series-3-Sum-of-Numbers.ts
export function getSum(a: number, b: number): number {
	const min = Math.min(a, b);
	const max = Math.max(a, b);
	let sum = 0;

	for (let index = min; index <= max; index++) {
		sum += index;
	}

	return sum;
}

console.log(getSum(-1, 2));
